spec/countries/MEX-spec.coffee

```diff
@@ -79,3 +79,23 @@ describe 'Mexico', ->
 
 			# Assert
 			expect(result).to.be.true
+
+		it 'all Mexico NDCs', ->
+			# Arrange
+			prefix  = "+52 "
+			suffix = " 111 2222"
+			suffix8 = "0 111 2222"
+
+			# Act
+			for ndc in Phone.countries['52'].nationalDestinationCode
+				if ndc.length is 2
+					number = prefix + ndc + suffix8
```

Comment:
@marcoskwkm Your opinion: is it too ugly? Any better idea? hahaha

Reply:
Looks fine to me.